datadog_checks_dev/datadog_checks/dev/env.py

```diff
@@ -45,16 +45,22 @@ def environment_run(up, down, sleep=None, endpoints=None, conditions=None, env_v
     result = None
     with env_vars, wrapper:
         try:
+            key = 'environment_result_{}'.format(up.__class__.__name__.lower())
             if set_up_env():
                 result = up()
+                set_env_vars({key: serialize_data(result).decode('utf-8')})
```

Comment:
This new logic releated to `environment_result_* ` seems not obvious at first glance, some doc could help.

Reply:
Added some comments.